kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/KyuubiScalaObjectMapper.scala

```diff
@@ -19,11 +19,13 @@ package org.apache.kyuubi.server.api
 
 import javax.ws.rs.ext.ContextResolver
 
-import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.{DeserializationFeature, ObjectMapper}
 import com.fasterxml.jackson.module.scala.DefaultScalaModule
 
 class KyuubiScalaObjectMapper extends ContextResolver[ObjectMapper] {
-  private val mapper = new ObjectMapper().registerModule(DefaultScalaModule)
+  private val mapper = new ObjectMapper()
+    .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+    .registerModule(DefaultScalaModule)
```

Comment:
How about using a chaining and fluent style builder with JsonMapper (since 2.10) ?
`JsonMapper` itself extends `ObjectMapper`in JSON-format specific.

```suggestion
  private val mapper = JsonMapper.builder()
    .addModules(DefaultScalaModule)
    .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
    .build()
```

Reply:
thanks for the suggestion.

Now there are multiple parts of code likes this. So, I think that we can just keep this as it is now.

Maybe you can raise a dedicated pr to refactor them all.

Reply:
SGTM.